src/config/graphical-session/wayland.md

```diff
@@ -26,6 +26,7 @@ Void Linux currently packages the following Wayland compositors:
 - Wayfire: 3D Wayland compositor
 - Hikari: a stacking compositor with some tiling features
 - Cage: a Wayland kiosk
+- River: a dynamic tiling Wayland compositor
```

Comment:
I appreciate the change, but this makes me think we should consider removing the list entirely.